examples/basic/scatters/image_url.py

```diff
@@ -1,3 +1,12 @@
+''' An scatter plot showing `Bokeh image logo`_ as marker.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.image_url
+    :refs: :ref:`ug_basic_scatters_urls`
```

Comment:
unfortunately this ref doesn't actually exist yet. Either it needs to be removed altogether, or else it must be added to the docs ReST source:

https://raw.githubusercontent.com/bokeh/bokeh/branch-3.2/docs/bokeh/source/docs/user_guide/basic/scatters.rst

It would go right above the section header: 

<img width="379" alt="Screenshot 2023-03-21 at 19 37 54" src="https://user-images.githubusercontent.com/1078448/226787731-5bbb241d-ae2d-43b5-805f-06817f4ac58a.png">

Unfortunately GitHub tries to "helpfully" render ReST files so I to link the raw file and show an image